clients/java/signalr/build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
     implementation 'com.google.code.gson:gson'
     implementation 'com.squareup.okhttp3:okhttp'
     implementation 'io.reactivex.rxjava2:rxjava'
+    implementation 'org.slf4j:slf4j-api'
```

Comment:
cc @Eilon for a new dependency

Reply:
FYI, for Eilon, the version number here is: 1.7.25

Version numbers were centralized in a common "bill of materials" package across Microsoft in https://github.com/aspnet/SignalR/pull/3114

Reply:
Go for it.